app/controllers/admin/announcements_controller.rb

```diff
@@ -40,7 +43,8 @@ def update
     params.require(:announcement).permit(:event_id, :title, :content, :status)
   end
 
-  private def push_notification_message(announcement)
+  # @rbs announcement: Announcement
+  private def push_notification_message(announcement) #: Hash[untyped, untyped]
```

Comment:
I choose using YARD style for arguments and `#:` style for return type.
RBS::Inline provides 3 styles to annotate types. And I prefer this style.

Please feel free to give your opinion :-)


refs: https://www.timedia.co.jp/tech/20241017-tech/

Reply:
FYI: This is the syntax guide for RBS::Inline.
https://github.com/soutaro/rbs-inline/wiki/Syntax-guide

Reply:
Finally I choose using full YARD style (including return type).